src/tools/dom-utils/autoinit/index.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-console */
+/* eslint-disable no-plusplus */
```

Comment:
Why adding an exclusion and not changing the syntax to prevent the error..? What was wrong with the previous forEach, by the way..?

Reply:
The  previous foreach was wrong, removing elements from the array during the loop misleads the final result. 
it's therefore necessary to make a reverse to avoid any breaking, and loop backwards..